### ISSUE TITLE
Need a better name than suppressParent

### ISSUE THREAD

**Issue by davidfowl** [NuGet/Home#2196]
I've been thinking about this for a while and I think we need a better name. This is to track suggestions for renames.
## Scenario 1: Runtime only dependencies

``` JSON
{
    "version": "1.0.0-*",
    "dependencies": {
        "Newtonsoft.Json": {
            "version": "8.0.2",
            "suppressParent": "compile"
        }
    },
    "frameworks": {
        "net45": { }
    }
}
```

`nuget pack`

``` XML
<dependencies>
  <group>
    <dependency id="Newtonsoft.Json" version="1.0.0" exclude="compile" />
  </group>
</dependencies>
```

The names `include` and `exclude` apply to the `project.json` and don't match the nuspec behavior. The
`suppressParent` property matches the `exclude` behavior of the nuspec. This is a little bit confusing.

/cc @emgarten

**Comment by yishaigalatzer** [comment]
This is already RTM.

**Comment by davidfowl** [comment]
Yes, we can deprecate things and support old and new but advertise the new.

**Comment by rrelyea** [comment]
current plan is to move to "privateAssets"

**Comment by rrelyea** [comment]
done. implemented in 3.6beta2 (preview 5 oob)
